.travis.yml

```diff
@@ -5,4 +5,4 @@ node_js:
 env:
 - TEST_DIR=.
 - TEST_DIR=scripts/babel-relay-plugin
-script: cd $TEST_DIR && npm install && npm test
+script: cd $TEST_DIR && npm install && npm run lint && npm test
```

Comment:
@zpao couldn't it be just something like:

```yml
script:
  - cd $TEST_DIR && npm install && npm test
  - npm run lint
```

`npm run lint` runs `eslint .` so it will check everything anyway (we would need to remove `/scripts` from `.eslintignore`)